storage/buckets/main.go

```diff
@@ -137,6 +137,40 @@ func list(client *storage.Client, projectID string) ([]string, error) {
 	return buckets, nil
 }
 
+func getBucketMetadata(w io.Writer, client *storage.Client, bucketName string) (error) {
+	ctx := context.Background()
+	// [START storage_get_bucket_metadata]
+	attrs, err := client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintln(w, "BucketName: %v", attrs.Name)
+	fmt.Fprintln(w, "Location: %v", attrs.Location)
+	fmt.Fprintln(w, "LocationType: %v", attrs.LocationType)
+	fmt.Fprintln(w, "StorageClass: %v", attrs.StorageClass)
+	fmt.Fprintln(w, "TimeCreated: %v", attrs.Created)
+	fmt.Fprintln(w, "Metageneration: %v", attrs.MetaGeneration)
+	fmt.Fprintln(w, "PredefinedACL: %v", attrs.PredefinedACL)
+	fmt.Fprintln(w, "DefaultKmsKeyName: %v", attrs.BucketEncryption.DefaultKMSKeyName)
+	fmt.Fprintln(w, "IndexPage: %v", attrs.BucketWebsite.MainPageSuffix)
+	fmt.Fprintln(w, "NotFoundPage: %v", attrs.BucketWebsite.NotFoundPage)
+	fmt.Fprintln(w, "DefaultEventBasedHold: %v", attrs.DefaultEventBasedHold)
+	fmt.Fprintln(w, "RetentionEffectiveTime: %v", attrs.RetentionPolicy.EffectiveTime)
+	fmt.Fprintln(w, "RetentionPeriod: %v", attrs.RetentionPolicy.RetentionPeriod)
+	fmt.Fprintln(w, "RetentionPolicyIsLocked: %v", attrs.RetentionPolicy.IsLocked)
+	fmt.Fprintln(w, "RequesterPays: %v", attrs.RequesterPays)
+	fmt.Fprintln(w, "VersioningEnabled: %v", attrs.VersioningEnabled)
+	fmt.Fprintln(w, "LogBucket: %v", attrs.BucketLogging.LogBucket)
+	fmt.Fprintln(w, "LogObjectPrefix: %v", attrs.BucketLogging.LogObjectPrefix)
+	fmt.Fprintln(w, "\n\n\nLabels:")
+	for key, value := range attrs.Labels {
+		fmt.Fprintln(w, "\t%v = %v", key, value)
+	}
+	// [END storage_get_bucket_metadata]
+	return nil
```

Comment:
nit: The other `get*` functions return the object as well - should you do that for this one?

Reply:
I instead verify the output of the sample in this case. Is that fair?

Reply:
Yeah, it's more consistent with the new format but let consistent with the samples around it to print instead. I'm just pointing it out, I'm don't think it makes a significant different either way

Reply:
I split it up. PTAL